supabase/client.py

```diff
@@ -100,7 +94,7 @@ def from_(self, table_name: str):
         )
         return query_builder.from_(table_name)
 
-    def rpc(self, fn, params):
+    def rpc(self, fn, params) -> Coroutine[Any, Any, Response]:
```

Comment:
```suggestion
    def rpc(self, fn: str, params: Dict[Any, Any]) -> Coroutine[Any, Any, Response]:
```
Missing parameter annotations